src/DisplayPlug.vala

```diff
@@ -208,6 +240,19 @@ public class Display.Plug : Switchboard.Plug {
         search_results.set ("%s → %s".printf (display_name, _("Screen mirroring")), "");
         return search_results;
     }
+
+    private void detect_touchscreen () {
```

Comment:
This function could be `static` since it doesn't access anything from the instance.